lib/screen/note_screen.dart

```diff
@@ -111,6 +113,13 @@ class _NoteScreen extends State<NoteScreen> with CommandHandler {
         ),
         titleSpacing: 0,
         elevation: 0,
+        actions: <Widget>[
```

Comment:
I will update this icon later by using the google fonts you added during your task